### ISSUE TITLE
Add list support to config yml parser

### ISSUE THREAD

**Issue by Ornias1993** [Ornias1993/jailman#104]
# Issue Template

### Detailed Bug Report
Currently the yml parser cant perse lists. However for features such as #90 it's important to make sure lists get parsed right.

### Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:  

1.  Make a list in the config file
2.  See it fail
3.

**Comment by Ornias1993** [comment]
@asmod3us This issue is one I think would be a good start starting to do thing in Python instead of bash, what do you think?

**Comment by asmod3us** [comment]
Yes, the current parser is not simple to extend... would be useful for mountpoints to have a proper yaml parser.

**Comment by Ornias1993** [comment]
In that case I would propose moving this to 1.4.0 instead...
I think thats a good place to add the first python...

**Comment by Ornias1993** [comment]
Actually it might even be good dropping those strange variables altogether and moving it to arrays (and lists) completely.
